bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -56,12 +57,49 @@ spec:
               exit 0
           }
           trap quit SIGTERM
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
-          ovs-appctl vlog/set "file:${OVS_LOG_LEVEL}"
-          /usr/share/openvswitch/scripts/ovs-ctl --protocol=udp --dport=6081 enable-protocol
+          chroot /host /bin/bash <<EOF
+          set -x
+          ln -s /run /var/run
+          #SYSTEMD_IGNORE_CHROOT=yes systemctl start ovsdb-server
+          export SYSTEMD_IGNORE_CHROOT=yes
+          echo "systemd ignore chroot set to " \$SYSTEMD_IGNORE_CHROOT
+          systemctl start ovsdb-server
+          if [[ $? == 0 ]]; then
```

Comment:
I think writing shell script without `set -e` or more generally [strict mode](http://redsymbol.net/articles/unofficial-bash-strict-mode/) is just asking for unreliability at best.  If e.g. the `ln -s` above failed for some reason the script just blindly stumbles on, and something *might* fail later.

We need to ensure that failures are observable and also propagate up into the Kuberentes/OpenShift layers.  See also https://github.com/openshift/machine-config-operator/issues/1365

I think you should use `set -e` in this whole script, and then drop testing `$?`.

Reply:
(Or, write this in a compiled language like Go/Rust)